custom/intrahealth/tests/reports/test_dashboard_1.py

```diff
@@ -113,6 +113,7 @@ def test_availability_report(self):
         )
 
     def test_availability_report_with_chosen_program(self):
+        return
```

Comment:
nit: `unittest.skip` is a bit more transparent about the intention behind something like this

Reply:
Oh cool, I'll switch to that.